<launch>
  <!-- No namespace here as we will share this description. 
       Access with slash at the beginning -->
  <param name="robot_description" command="$(find xacro)/xacro --inorder $(find turtlebot3_description)/urdf/turtlebot3_burger_for_autorace.urdf.xacro" />


  <!-- BEGIN ROBOT 1-->
  <group ns="robot1">
    <param name="tf_prefix" value="robot1_tf" />
    <include file="$(find swarmbots)/launch/one_robot.launch" >
    <arg name="model" value="burger"/>
      <arg name="x_pos" value="1" />
      <arg name="y_pos" value="1" />
      <arg name="z_pos" value="0" />
      <arg name="robot_name"  value="Robot1" />
    </include>
  </group>

  <!-- BEGIN ROBOT 2-->
  <group ns="robot2">
    <param name="tf_prefix" value="robot2_tf" />
    <include file="$(find swarmbots)/launch/one_robot.launch" >
    <arg name="model" value="burger"/>
      <arg name="x_pos" value="-1" />
      <arg name="y_pos" value="1" />
      <arg name="z_pos" value="0" />
      <arg name="robot_name"  value="Robot2" />
    </include>
  </group>
</launch>